packages/react-server/core/common.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
 	RootContainer: require("./components/RootContainer"),
 	RootElement: require("./components/RootElement"),
 	Link: require('./components/Link'),
+	TheFold: require('./components/TheFold').default,
```

Comment:
Awkward to pull out `.default` here, but IMO worth it for the cleaner implementation in `TheFold.js`.